inc/class-custom-walker-nav-menu.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/**
+ * Navigation menu walker class
+ *
+ * @package uds-wordpress-theme
+ */
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/*
+ * Class Name: Custom_Walker_Nav_Menu
```

Comment:
Please rename class: Uds_Custom_Walker_Widget_Nav_Menu

Reply:
Is there a naming standards to follow? or is it a suggestion to change the class name to Uds_Custom_Walker_Widget_Nav_Menu?

Reply:
It is more the case that I was trying to make the class name more explicit that this class generates ASU Web Standards nav menus.

As this [PHP wiki page](https://wiki.php.net/rfc/class-naming) shows, there is friction in the PHP community about class naming conventions. I usually follow the "Always Pascal" rule, which means the first character is capitalized and all following letters are always lowercase. So that is why I asked for "Uds_Custom..." instead of "UDS_Custom...".

I am fine with the second option if you want to use that. I just suggested the usual naming schema we have used in our other projects.

Reply:
No problem, I'll change it to Uds_Custom_Walker_Widget_Nav_Menu